### ISSUE TITLE
[BUG] Engine Master switch on Throttle causes Speed and HDG issue

### ISSUE THREAD

**Issue by andrejetcat** [flybywiresim/a32nx#2482]
**Mod Version**

master FBW

**Describe the bug**

i have the thrustmaster Airbus Throttle connected, with latest firmeware and proper set with the FBW mod. when I start the engines "manually" in the virtual cockpit, i don´t have any issues.
When i select the ENG1 & 2 start switches on the Thrustmaster throttle (set to toggle fuel valves), also the switch in the cockpit work, and the engine start, like it shoud BUT thereafter i could only select ten´s values on the speed and HDG selector. once i switch off the physical buttons on the Throttle, it works normal. 
hard to explain, i know. It seems that the problem is present, when a button is connected with the "fuel valve" in the controller settings. when the fuel valve (eng1/2 start switch) is turned on by mouse in the cockpit, the issue isen´t there.

**To Reproduce**

1. start engine with thrustmaster airbus throttle (or any other, but configured with "fuel valve on/off"  -after that try to select a proper heading in ONE steps - not possible
2. same procedure inly by clicking "virtual"  - no issues
3.

**Expected behavior**

should work like if the buttons are clicked "virtual"

**Actual behavior**

if a "fuel valve" button is assigned with physical button on the (e.g. thrustmaster airbus throttle) the virtual switch does also work proper, but the discribed issues is now there.

**References**

hard to show - hope the discribtion helps - please ask if it is unclear

**Additional context**

Was this working before/when did the issue start occurring?
i don´t know
Is this a problem in the vanilla unmodded game?
i don´t know

Discord username (if different from GitHub):

**Comment by Sirius1708** [comment]
I have the same issue (also using the Thrustmaster TCA throttle) and can confirm the description.
Using the physical engine switches on the throttle causes the problem, when using the virtual/in-game switches everything works fine.

Button 3 (left engine start switch) is assigned to "Set Engine Master 1", "Set Starter 1", "Set Engine 1 Fuel Valve", "Toggle Engine 1 Fuel Valve"; Button 4 (right engine start switch) is set respectively for engine number 2.

[https://imgur.com/qN4bHGQ](url)

**Comment by andrejetcat** [comment]
I hope it can be fixed... I don´t know if it is a FBW issue. Its sad, that the Throttle Quadrant´s buttons are mainly useless with this issue.

**Comment by Sirius1708** [comment]
> I hope it can be fixed... I don´t know if it is a FBW issue. Its sad, that the Throttle Quadrant´s buttons are mainly useless with this issue.

I read in the MSFS forums that it's also possible to manage the speed/heading selectors with mouseclicks on the knobs.
Tried it and it worked after using the TCA throttle engine switches before. So the issue seems also to be related with using the mousewheel to change the speed/heading. Mouseclicks are less comfortable but it's ok as long as it's working.

**Comment by Saschl** [comment]
It is possibly related to this: https://forums.flightsimulator.com/t/heading-increment-bug-10-degree-instead-of-1-explained/290173/105

tldr; any control that causes constant input (eg the engine master switch) will somehow cause this bug. It seems related to the msfs core and I don’t know if it can be fixed by fbw). If I got time I might try looking into the issue.

**Comment by Saschl** [comment]
This issue should not appear anymore,  WU4 fixed this bug.

**Comment by tracernz** [comment]
This is fixed in the sim.
